areas/VirtualDesktop/src/AzureMcp.VirtualDesktop/Options/VirtualDesktopOptionDefinitions.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace AzureMcp.Areas.VirtualDesktop.Options;
+
+public static class VirtualDesktopOptionDefinitions
+{
+    public const string HostPoolName = "hostpool-name";
+    public const string HostPoolResourceId = "hostpool-resource-id";
+    public const string SessionHostName = "sessionhost-name";
```

Comment:
Can we remove -name from sessionhost and hostpool?  We had some inconsistencies and decided to not use -name unless necessary.